src/odemis/acq/test/fastem_conf_test.py

```diff
@@ -95,6 +101,10 @@ def test_configure_scanner_live(self):
         self.assertTrue(self.scanner.immersion.value)
         self.assertEqual(self.scanner.rotation.value, math.radians(5))
 
+        scanner_md = self.scanner.getMetadata()
+        # check that the MD_POS_COR is set to [0, 0] for live stream imaging.
+        self.assertListEqual([0, 0], scanner_md[model.MD_POS_COR])
+
```

Comment:
I think this test should also be part in the `configure_scanner_megafield` test to make sure that it is also set correctly in this mode.